ui/index.go

```diff
@@ -0,0 +1,82 @@
+// Copyright © 2018 Thomas Winsnes <tom@vibrato.com.au>
```

Comment:
this license should probably be switched over to Vibrato... assuming @TWinsnes is ok with that.

Reply:
I'm okay with that